contrib/pax_storage/src/cpp/catalog/pax_aux_table.cc

```diff
@@ -278,6 +287,11 @@ void UpdateVisimap(Oid aux_relid, int block_id, const char *visimap_filename) {
         NameGetDatum(&pt_visimap_name);
   }
 
+  // visimap only update in delete operation, and stats is always valid
+  repls[ANUM_PG_PAX_BLOCK_TABLES_PTISSTATSVALID - 1] = true;
+  nulls[ANUM_PG_PAX_BLOCK_TABLES_PTISSTATSVALID - 1] = false;
+  values[ANUM_PG_PAX_BLOCK_TABLES_PTISSTATSVALID - 1] = BoolGetDatum(true);
```

Comment:
Why is `isstatsvalid` changed if the visimap file is updated? If it's not changed, repls[..] should be false and leave nulls[..] and values[..] unset.